scripts/forward.py

```diff
@@ -2,90 +2,122 @@
 PKG = 'embodied_attention'
 import roslib; roslib.load_manifest(PKG)
 
+import rospy
 import cv2 as cv
 import numpy as np
-import skimage.transform
-from keras.models import load_model
-import rospy
+import tensorflow as tf
+import matplotlib.pyplot as plt
 from sensor_msgs.msg import Image, CameraInfo
 from cv_bridge import CvBridge, CvBridgeError
-import sys
-import os
 
-height = 168
-width = 224
+# network has been trained and works best on 192x256
+# both height and width must be divisible by 8
+img_height = 192
+img_width = 256
 
-def rescale(img, height, width):
-    if len(img.shape) == 3:
-        h, w, _ = img.shape
-    elif len(img.shape) == 2:
-        h, w = img.shape
+def rescale_image(img, tar_h, tar_w):
```

Comment:
abbreviation "tar" for target is not obvious, I would call it:
`def rescale_image(img, new_h, new_w):`